newIDE/app/src/EventsSheet/ParameterFields/LeaderboardIdField.js

```diff
@@ -122,8 +135,8 @@ export function LeaderboardIdField(props: ParameterFieldProps) {
               buildMenuTemplate={i18n => [
                 {
                   label: isTextInput
-                    ? i18n._(t`Switch to expression`)
-                    : i18n._(t`Switch to select`),
+                    ? i18n._(t`Select the leaderboard from a list`)
+                    : i18n._(t`Enter the leaderboard id as an expression`),
```

Comment:
I'm not sure I understand what this means.
If this is for the user to enter the id themselves, should this be more `type the leaderboard ID manually`?